tests/ixia/files/helper.py

```diff
@@ -0,0 +1,16 @@
+from tests.common.broadcom_data import is_broadcom_device
+from tests.common.helpers.assertions import pytest_require
+
+def skip_warm_reboot_td2(duthost, reboot_type):
```

Comment:
it seems to me that this function should be more general, e.g., skip_warm_reboot, and use asic_type as an argument.

Reply:
Warm reboot support is not available only on this asic. I can rename the function to "skip_warm_reboot". In future if we want to skip for more asics, we can then just update this function to include that asic as well in the skip list

Reply:
> Warm reboot support is not available only on this asic. I can rename the function to "skip_warm_reboot". In future if we want to skip for more asics, we can then just update this function to include that asic as well in the skip list

Let's rename this function to skip_warm_reboot. Inside the function, we can have a list of asics that do not support warm reboot.